apps/bfd-pipeline/bfd-pipeline-rda-grpc/pom.xml

```diff
@@ -17,10 +17,10 @@
 
     <properties>
         <!-- Version of gRPC related dependencies -->
-        <grpc_java.version>1.37.0</grpc_java.version>
+        <grpc_java.version>1.47.0</grpc_java.version>
         <!-- Version of the plugin used to build the gRPC stubs from the Health.proto file. -->
-        <protoc.version>3.15.8</protoc.version>
-        <protobuf.version>3.12.0</protobuf.version>
+        <protoc.version>3.21.2</protoc.version>
+        <protobuf.version>3.21.2</protobuf.version>
```

Comment:
how about pulling these properties up to the parent so that we only have them in one place?

Reply:
Yeah I'm ok with that.  I think it started off this way just because we didn't initially plan on creating bfd-pipeline-rda-grpc-apps, but now that both pom.xml files need it that makes sense.  @brianburton thoughts?